projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -1,20 +1,100 @@
-import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	__experimentalInputControl as InputControl, // eslint-disable-line wpcalypso/no-unsafe-wp-apis
+	SelectControl,
+} from '@wordpress/components';
+import { useCallback, useEffect, useState } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { find, includes, map } from 'lodash';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
As we're starting from scratch here, I'm wondering if we should consider using `@wordpress/data` over `redux` directly, but that's just something to think about now.

Reply:
That's what I would recommend as well. This is what we use elsewhere.

Reply:
Everything has been now moved to wordpress/data